key-vault.tf

```diff
@@ -1,16 +1,16 @@
 module "bar-vault" {
-  source = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
-  name = "${var.product}-${var.env}"
-  product = "${var.product}"
-  env = "${var.env}"
-  tenant_id = "${var.tenant_id}"
-  object_id = "${var.jenkins_AAD_objectId}"
-  resource_group_name = "${azurerm_resource_group.rg.name}"
+  source = "git@github.com:hmcts/cnp-module-key-vault?ref=azurermv2"
+  name = join("-", [var.product, var.env])
+  product = var.product
```

Comment:
Replace var.product with **join("-", [var.product, "shared"])**

Reply:
we are using the same rg name